ios/RNFun.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This looks like stuff that got accidentally committed

Reply:
looks like it's safe to commit, i'm ok with it

see also https://stackoverflow.com/a/50368149 re: this file

Reply:
actually, reading the contents of this file esp things about showing a warning or w/e, this might be good to put in `.gitignore`

Reply:
im v conflicted, i think this is fine lol